plugins/out_es/es.c

```diff
@@ -75,6 +75,27 @@ static flb_sds_t add_aws_auth(struct flb_http_client *c,
 }
 #endif /* FLB_HAVE_AWS */
 
+static int es_http_add_cloud_apikey(struct flb_http_client *c,
```

Comment:
There seems to be some opportunities for refactoring http auths in the `flb_http_client` library, but should probably be tackled in another PR.